src/ledger/LedgerManagerImpl.cpp

```diff
@@ -561,7 +561,8 @@ LedgerManagerImpl::loadLastKnownLedger(bool restoreBucketlist)
     // (standalone offline commands, in-memory setup) do not need to
     // spin up expensive merge processes.
     auto assumeStateWork = mApp.getWorkScheduler().executeWork<AssumeStateWork>(
-        has, latestLedgerHeader->ledgerVersion, restoreBucketlist);
+        has, latestLedgerHeader->ledgerVersion,
+        /*restoreBucketlist*/ fullLoad);
```

Comment:
The new name seems a bit confusing. Maybe it'd be clearer to flip the flag and name it `loadLCLOnly` (as opposed to full state) or something like that?

Reply:
I've renamed this to `skipBuildingFullState` in order to emphasize that we're skipping something. I find `loadLCLOnly` confusing as well, as it's not obvious what exactly comprises LCL. This flag also might be used more generically, in case if we have more 'optional' state that we can sometimes skip. I've also added a comment.